jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/HttpContent.java

```diff
@@ -78,4 +84,136 @@
          */
         HttpContent getContent(String path) throws IOException;
     }
+
+    /**
+     * HttpContent Wrapper.
+     */
+    class HttpContentWrapper implements HttpContent
```

Comment:
Just call it `Wrapper` like all the others.

Reply:
done